### DISCUSSION TITLE
Where to ask for features?

### DISCUSSION THREAD

**Discussion by senpro-ingwersenk** [dani-garcia/vaultwarden#4717] [General]
Hello there!

I have been using Vaultwarden both privately and within this company I work at - and especially the ladder has ran into some interesting limits regarding management of users and orgs. So far I have filed two "tickets"; one here in the [discussions](https://github.com/dani-garcia/vaultwarden/discussions/4701) and another [in the forum](https://vaultwarden.discourse.group/t/feature-request-user-and-org-management-in-batches/3749).

I am keenly aware how much work goes into Vaultwarden; but before I go and drop feature requests into the void, I just wanted to know what the "best practice" is. :)

Thank you very much and kind regards!

**Comment by BlackDex** [comment]
Mostly, be patient on the one post and maybe bump that specific thread. Not creating multiple posts referring to each of them.

It is only confusing and doesn't serve any purpose in my opinion.
If anything x i would suggest to join the chat room and ask there what people think about the first post instead of multiple other posts.

**Reply by senpro-ingwersenk** [reply]
Got it, thank you!
